packages/screens/Settings/components/UserProfileModal.tsx

```diff
@@ -0,0 +1,325 @@
+      fontMedium14,
+      {
+        marginHorizontal: "auto",
+        color: neutralA3,
+        marginTop: layout.padding_x1,
+        width: width < smallMobileWidth ? 0.9 * width : 15 * layout.padding_x4,
+      },
+    ]),
+    cardContainer: {
+      display: "flex",
+      flexDirection: "row",
+      width: "100%",
+      marginHorizontal: "auto",
+      justifyContent: "center",
+      alignItems: "center",
+      gap: layout.padding_x1_5,
+      marginTop:
+        width < mobileWidth
+          ? width < smallMobileWidth
+            ? 5 * layout.padding_x0_5
+            : 30 * layout.padding_x0_5
+          : 45 * layout.padding_x0_5,
+    },
+    unselectedCardBox: {
+      display: "flex",
+      flexDirection: "row",
+      justifyContent: "center",
+      alignItems: "center",
+      width: childrenWidth,
+      paddingVertical: layout.padding_x1_5,
+      borderColor: neutral33,
+      borderWidth: 1,
+      backgroundColor: neutral00,
+      borderRadius: layout.padding_x1,
+      position: "relative",
+    },
+    selectedCardBox: {
+      display: "flex",
+      flexDirection: "row",
+      justifyContent: "center",
+      alignItems: "center",
+      width: childrenWidth,
+      paddingVertical: layout.padding_x1_5,
+      borderColor: neutral33,
+      borderWidth: 1,
+      backgroundColor: neutral17,
+      borderRadius: layout.padding_x1,
+      position: "relative",
+    },
+    confirmedCardBox: {
+      display: "flex",
+      flexDirection: "row",
+      justifyContent: "center",
+      alignItems: "center",
+      width: childrenWidth,
+      paddingVertical: layout.padding_x1_5,
+      borderColor: "rgba(0, 0, 0, 0.9)",
+      borderWidth: 1,
+      backgroundColor: "rgba(0, 0, 0, 0.9)",
+      borderRadius: layout.padding_x1,
+      position: "relative",
+    },
+    unconfirmedCardContent: StyleSheet.flatten([fontSemibold14]),
+    confirmedCardContent: StyleSheet.flatten([
+      fontMedium14,
+      {
+        color: neutral55,
+      },
+    ]),
+    confirmButton: {
+      position: "absolute",
+      top: -50,
+      left: childrenWidth / 2 - 20,
+      width: 60,
+      height: 60,
+      zIndex: 999,
+    },
+    buttonGroup: {
+      display: "flex",
+      justifyContent: "center",
+      flexDirection: "row",
+      gap: layout.padding_x3,
+      marginTop:
+        width < mobileWidth
+          ? width < smallMobileWidth
+            ? 5 * layout.padding_x0_5
+            : 30 * layout.padding_x0_5
+          : 45 * layout.padding_x0_5,
+    },
+  });
+
+  const updateConfirmedData = (item: any) => {
+    if (!confirmedData.includes(item.content))
+      setConfirmedData([...confirmedData, item.content]);
+    else {
+      const targetIndex = confirmedData.indexOf(item.content);
+      const targetData = confirmedData;
+      targetData.splice(targetIndex, 1);
+      setConfirmedData(targetData);
+    }
+  };
+
+  const renderItem = (item: any, index: any) => {
+    return (
+      <Hoverable
```

Comment:
Note: `Hoverable` is required to have "onMouseEnter", as `View` don't allow this.